reason-react-native/src/apis/Linking.md

```diff
@@ -1,50 +1,112 @@
 ---
 id: apis/Linking
 title: Linking
-wip: true
+officialDoc: https://facebook.github.io/react-native/docs/linking.html
 ---
 
+Only for projects supporting Native Code. Requires the project to have been
+created using `react-native init` or ejected afterwards if created using
+`expo init` or `create-react-native-app`.
+
+## Methods
+
+### `openURL`
+
+To attempt opening a URL. URL should be specified as `string`, and the method
+returns `unit` wrapped in a promise, that is `Js.Promise.t(unit)`. Promise is
+resolved if the user approves the request (through the _open dialog_) or the
+link is opened automatically. If the user rejects the request or there are no
+registered applications supporting the URL, the promise is rejected. It is
+recommended to use the `canOpenURL` method beforehand, to verify that the URL
+can indeed be opened.
+
 ```reason
-type extra;
+openURL(string)
```

Comment:
I was wondering if we should use real world snippet by including module name.
That would mean:

```reason
open ReactNative;

Linking.openURL("http://some.url")
```

We could potentially omit `open ReactNative;` as it like a requirement (at least I guess we are going to promote this usage), but we should probably show something that people can easily copy/paste/tweak without too much thing to think about.

What do you think?

Reply:
Sure, I'll create an example. Since it wouldn't hurt, I'll keep the open statement to make it as self-contained as possible.

Reply:
I revised to list the type for each method, considering that the example should be sufficient to demonstrate how the methods can be used.

Should we nevertheless have examples of how each method is called, as in the example you have given for `openURL`?

Reply:
I would do that for all code blocks indeed. I found this much more clear than a "raw api" declaration in code as you can directly identify how you are going to use it.
@cknitt thoughts?

Reply:
ORRR maybe we should have both like this

- code showing api, very concise & with type, not values (exactly what you did) 
- description blah blah blah
- very tiny example (like the one I shared)

And a "real world" example at the end, mixing multiple usage if possible (exactly what you did).